lookout/style/typos/corruption.py

```diff
@@ -0,0 +1,95 @@
+import random
+import string
+
+import pandas
+from tqdm import tqdm
+
+from lookout.style.typos.utils import Columns
+
+
+letters = list(string.ascii_lowercase)
+
+
+def rand_insert(token: str) -> str:
+    """
+    Add random letter inside a token.
```

Comment:
```suggestion
    Add a random letter inside a token.
```
(and all the places below)